python_apps/pypo/pypo/pypoliquidsoap.py

```diff
@@ -120,13 +120,12 @@ def verify_correct_present_media(self, scheduled_now):
 
         try:
             scheduled_now_files = \
-                    filter(lambda x: x["type"] == eventtypes.FILE, scheduled_now)
+                    [x for x in scheduled_now if x["type"] == eventtypes.FILE]
 
             scheduled_now_webstream = \
-                    filter(lambda x: x["type"] == eventtypes.STREAM_OUTPUT_START, \
-                            scheduled_now)
+                    [x for x in scheduled_now if x["type"] == eventtypes.STREAM_OUTPUT_START]
 
-            schedule_ids = set(map(lambda x: x["row_id"], scheduled_now_files))
+            schedule_ids = set([x["row_id"] for x in scheduled_now_files])
```

Comment:
You can use a set-comprehension directly:

```suggestion
            schedule_ids = {x["row_id"] for x in scheduled_now_files}
```